src/main/java/lotto/view/ResultView.java

```diff
@@ -19,14 +22,23 @@
 
   public static void printBuying(List<LottoGame> lottoGames) {
     lottoGames.stream()
-        .map(LottoGame::getLottoNos)
-        .forEach(lottoNos -> System.out.println(lottoNos.stream().map(LottoNo::toString).collect(Collectors.joining(", ", "[", "]"))));
+        .map(LottoGame::getLottoBalls)
+        .forEach(lottoNos -> System.out.println(
+            lottoNos.stream()
+                .map(LottoBall::toString)
+                .collect(Collectors.joining(", ", "[", "]"))
+        ));
   }
```

Comment:
```suggestion
    public static void printBuying(List<LottoGame> lottoGames) {
        lottoGames.stream()
                .map(LottoGame::getLottoNos)
                .map(ResultView::formatResult)
                .forEach(System.out::println);
    }

    private static String formatResult(Set<LottoBall> lottoNos) {
        return lottoNos.stream()
                .map(LottoBall::toString)
                .collect(joining(DELIMITER_COMMA, PREFIX, SUFFIX));
    }
```

메서드를 추출하여 의도를 분명히 드러내는 것은 어떨까요.